src/System.Runtime.Numerics/src/System/Numerics/BigIntegerCalculator.Buffer.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System.Diagnostics;
```

Comment:
I'd prefer `BigIntegerCalculator.BitsBuffer.cs` instead of `BitsBuffer.cs` (lexicographical reasons). Since the other files have just abbreviations as suffix, I've chosen the shorter `BigIntegerCalculator.Buffer.cs` (long file names are so java...).

But I haven't that much emotion for these file names, just say and I amend. ;)